src/main/webapp/i18n/en/quizExercise.json

```
@@ -141,7 +141,7 @@
             "preview": "Preview Quiz",
             "solution": "Sample Solution",
             "quizIsOverText": "The quiz has ended. Please wait. If the results do not load automatically within 60s, please refresh the page.",
-            "notParticipatedText": "The quiz has ended and you did not participate. There are no results to display.",
+            "notParticipatedText": "The quiz has ended, and you did not participate. There are no results to display.",
```

```suggestion
            "notParticipatedText": "The quiz has ended and you did not participate. There are no results to display.",
```
preference: I would not put a comma here.